packages/form/src/types/input.ts

```diff
@@ -15,6 +17,8 @@ export type UiInputProps = {
   placeholder?: string;
   /* React ref */
   ref?: React.Ref<HTMLInputElement>;
+  /** Input field state */
+  state?: 'POSITIVE' | 'NEGATIVE' | 'ERROR';
```

Comment:
I'm not quite sure if `state` or `theme` for this prop.... 🤔

Reply:
Using `theme` might get confusing with the overall theme that is active in the project so I went ahead with `state` maybe is it fine?